keras/src/ops/numpy.py

```diff
@@ -3114,6 +3114,41 @@ def empty(shape, dtype=None):
     return backend.numpy.empty(shape, dtype=dtype)
 
 
+class EmptyLike(Operation):
+    def call(self, x, dtype=None):
+        return backend.numpy.empty_like(x, dtype=dtype)
+
+    def compute_output_spec(self, x, dtype=None):
+        dtype = dtype or x.dtype
+        dtype = backend.standardize_dtype(dtype)
+        return KerasTensor(x.shape, dtype=dtype)
```

Comment:
dtype is a parameter to `__init__` not `call`. Also, copying for the other ones:

```python
class EmptyLike(Operation):
    def __init__(self, dtype=None, *, name=None):
        super().__init__(name=name)
        self.dtype = None if dtype is None else backend.standardize_dtype(dtype)

    def call(self, x):
        return backend.numpy.empty_like(x, dtype=self.dtype)

    def compute_output_spec(self, x):
        dtype = (
            backend.standardize_dtype(x.dtype)
            if self.dtype is None
            else self.dtype
        )
        return KerasTensor(x.shape, dtype=dtype)
```